tox.ini

```
@@ -26,6 +26,12 @@ basepython = python3
 deps = flake8
 commands = flake8 {posargs:}
 
+[testenv:mypy]
+description = Check code style
+basepython = python3
+deps = mypy
+commands = mypy --ignore-missing-imports {posargs:.}
```

I'm thinking about moving the option into `posargs` placeholder:

```suggestion
commands = mypy {posargs: --ignore-missing-imports .}
```

Not sure if this would be helpful... :thinking: